docs/examples/webpack/webpack.config.js

```diff
@@ -83,25 +100,29 @@ module.exports = ({ WEBPACK_SERVE }, { mode }) => ({
 
   output: {
     clean: true,
-    filename: 'assets/javascripts/[name].min.js',
+    filename: 'javascripts/[name].min.js',
     iife: true,
     path: destPath,
     publicPath: '/'
   },
 
   plugins: [
+    new MiniCssExtractPlugin({
+      filename: 'stylesheets/[name].min.css'
+    }),
     new CopyPlugin({
       patterns: [
-        join(srcPath, 'index.html'),
-        {
-          context: join(frontendPath, './src/govuk/assets'),
-          from: '{fonts,images}/**',
```

Comment:
Webpack `css-loader` now automatically scans `url()` etc to extract the necessary assets

Previously we copied everything